### ISSUE TITLE
To install in doc root or parent?

### ISSUE THREAD

**Issue by jerry1970** [devvoh/parable#13]
With a dir `public` I assumed the project root (with dirs `app`, `public`, and `vendor`) was one level above the document root. (I prefer the app outside the document root.) With a default document root `public_html` I made a symlink from `public_html` to `public`. Works.

But now, `buildBaseUrl` (in `vendor/devvoh/parable/src/Http/Url.php`), removing `/public/index.php` from `$_SERVER['SCRIPT_NAME']`, results in a URL starting with the domain plus `/index.php/`, for example:
```
<link rel="icon" type="image/png" href="http://mytestdomain.dev/index.php/images/icon-32.png">
```

Easy to change but wanted to let you know.

**Comment by devvoh** [comment]
Good find, and a use case I hadn't considered just yet!

I'll make the public_folder configurable.

**Comment by devvoh** [comment]
Should be fixed in 0.12.5. Reopen if it requires fixes!
